test/parallel/test-fs-promises.js

```diff
@@ -398,6 +399,24 @@ async function getHandle(dest) {
       );
     }
 
+    // Create a file with `mkstemp`.
```

Comment:
Ideally we'd have a lot more tests here (for prefix behavior, prefix with X, etc) but I'm fine with that being a TODO for now.